05.wc/wc.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'optparse'
+
+def main
+  option, file_names = parse_options
+
+  if file_names.empty?
+    print_counts_for_stdin(option)
+  else
+    count_and_print_files(file_names, option)
+  end
+end
+
+def parse_options
+  opt = OptionParser.new
+  option = {}
+  opt.on('-l') { |l| option[:l] = l }
+  opt.on('-w') { |w| option[:w] = w }
+  opt.on('-c') { |c| option[:c] = c }
+  file_names = opt.parse(ARGV)
+  [option, file_names]
+end
+
+def print_counts_for_stdin(option)
+  file_content = $stdin.read
+  line, word, byte = count(file_content)
```

Comment:
```suggestion
  line_count, word_count, byte = count(file_content)
```

件数（数値）であることがわかるようにcountを付けるといいかも。
byteだけは数値であることが伝わりそうですが。